automationspn/script.ps1

```diff
@@ -0,0 +1,29 @@
+#Create a local function project
+#Choose folder name
+$foldername = Read-Host -Prompt "Enter a name for your folder" 
+
+#create local function project in powershell
+func init $foldername --powershell
```

Comment:
This part actually creates a function (and generates code) on your local machine. This is a separate task. You should remove that